tests/DCPS/Compiler/is_topic_type/more_is_topic_type.idl

```diff
@@ -0,0 +1,80 @@
+@default_nested
```

Comment:
Using `@default_nested` on the modules is the common factor, how about naming the file after that?